packages/liquidator/index.js

```diff
@@ -305,14 +336,16 @@ async function Poll(callback) {
       priceFeedConfig: process.env.PRICE_FEED_CONFIG ? JSON.parse(process.env.PRICE_FEED_CONFIG) : null,
       // If there is a liquidator config, add it. Else, set to null. This config contains crThreshold,liquidationDeadline,
       // liquidationMinPrice, txnGasLimit & logOverrides. Example config:
-      // {"crThreshold":0.02,  -> Liquidate if a positions collateral falls more than this % below the min CR requirement
+      // { "crThreshold":0.02,  -> Liquidate if a positions collateral falls more than this % below the min CR requirement
       //   "liquidationDeadline":300, -> Aborts if the transaction is mined this amount of time after the last update
       //   "liquidationMinPrice":0, -> Aborts if the amount of collateral in the position per token is below this ratio
       //   "txnGasLimit":9000000 -> Gas limit to set for sending on-chain transactions.
       //   "whaleDefenseFundWei": undefined -> Amount of tokens to set aside for withdraw delay defense in case position cant be liquidated.
       //   "defenseActivationPercent": undefined -> How far along a withdraw must be in % before defense strategy kicks in.
-      //   "logOverrides":{"positionLiquidated":"warn"}} -> override specific events log levels.
-      liquidatorConfig: process.env.LIQUIDATOR_CONFIG ? JSON.parse(process.env.LIQUIDATOR_CONFIG) : null,
+      //   "logOverrides":{"positionLiquidated":"warn"}, -> override specific events log levels.
+      //   "contractType":"ExpiringMultiParty", -> override the kind of contract the liquidator is pointing at.
+      //   "contractVersion":"1.2.2"":"ExpiringMultiParty"} -> override the contract version the liquidator is pointing at.
+      liquidatorConfig: process.env.LIQUIDATOR_CONFIG ? JSON.parse(process.env.LIQUIDATOR_CONFIG) : {},
```

Comment:
Doesn't this switching of the default value from `null` to `{}` remove the need for [this](https://github.com/UMAprotocol/protocol/pull/2394/files#diff-2eaa6768c5ce2a0129aa4b22dfb8f7f33c0cedd7bbb4cb7120ee3327ced9c4ebR71) change?

Reply:
you could still put null as the process variable. this change might be redundant though seeing it is cast on the other side. will test it a bit and get back to you.